### PR TITLE
Sdutta/targets

### PR DESCRIPTION
**Pull request by shduttacheezit**

### CODE REVIEW

**Comment by jburchard** [comment]
This is really great. It looks like it's failing the tests due to pep.

You can see them if you run `flake8 --max-line-length=100 parsons`

**Comment by shduttacheezit** [comment]
removed some linting that was in the error stack trace!

File: parsons/ngpvan/target_export.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    def get_target_export(self, export_job_id):
+        """
+        Get specific target export job id's status.
+
+        `Returns:`
+            Parsons Table
+                See :ref:`parsons-table` for output options.
+        """
+
+        tbl = Table(self.connection.get_request(f'targetExportJobs/{export_job_id}'))
```
[the remark this answers is not in the document]
Comment (eliotst) [reply parsons/ngpvan/target_export.py:24]: I think if the job is `Pending`/`InProcess`, you can return `None`. If its status is `Error`, I think you could raise an exception.

File: parsons/ngpvan/target_export.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
+"""NGPVAN Target Endpoints"""
+
+from parsons.etl.table import Table
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class TargetExport(object):
```
[the remark this answers is not in the document]
Comment (shduttacheezit) [reply parsons/ngpvan/target_export.py]: changes submitted!

File: parsons/ngpvan/target_export.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (shduttacheezit) [reply parsons/ngpvan/target_export.py]: changes submitted!

File: parsons/ngpvan/targets.py [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+            csv = json_obj[0]['file']['downloadUrl']
+            response_csv = requests.get(csv)
+            print(response_csv)
+            return Table.from_csv_string(response_csv.text)
+        elif job_status == 'Pending' or job_status == 'InProcess':
+            logger.info(f'Target export job is pending or in process for {export_job_id}.')
+        else:
+            raise TargetsFailed(f'Target export failed for {export_job_id}')
+
+        # tbl = Table(self.connection.get_request(f'targetExportJobs/{export_job_id}'))
```
Comment (eliotst) [inline parsons/ngpvan/targets.py:76]: Would you clean up this commented out code?

File: test/test_van/test_targets.py [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+        # Test that it doesn't throw and error
+        r = self.van.create_target_export(target_id, webhook_url=None)
+
+        self.assertEqual(r, export_job_id)
+
+
+    @requests_mock.Mocker()
+    def test_get_target_export(self, m):
+
+        # # Create response
```
Comment (eliotst) [inline test/test_van/test_targets.py:91]: More commented out code that could be cleaned out.

**Pull request merged**